src/autogpt_plugins/twitter/README.md

```diff
@@ -1,4 +1,4 @@
-# autogpt-twitter 🐣
+# desojo/autogpt-twitter 🐣
```

Comment:
I'd add your name as a maintainer somewhere lower so people don't ping you every 3 seconds asking for help :)

Reply:
Don't ask how I know

Reply:
Oh I'm fine with the visibility lol, we could also add maintainers as a column next to the plugin name within the base README.md if you'd rather

Reply:
With how the last PR was merged due to me being a git noob, I'd like to at least be listed as a contributor through this readme update if anything 😄

Reply:
yeah add it to the root  readme table